v20.2/performance-benchmarking-with-tpc-c-100k-warehouses.md

```diff
@@ -202,108 +209,24 @@ CockroachDB offers a pre-built `workload` binary for Linux that includes the TPC
     {% include copy-clipboard.html %}
     ~~~ shell
     $ ./workload fixtures import tpcc \
-    --warehouses 100000 \
+    --warehouses 150000 \
+    --partitions 81 \
+    --partition-strategy=leases
     "postgres://root@<address of any CockroachDB node>:26257?sslmode=disable"
     ~~~
 
-    This will load 8TB of data for 100,000 "warehouses". This can take around 6 hours to complete.
+    This will load the data for 150,000 warehouses. This can take several hours to complete.
```

Comment:
I went less specific here because I don't have the info re: data size, and how long it took.  If y'all can share that info I will add it back.

Normally I'd prefer to be a little less specific so the doc is easier to maintain but I think if a user is investing the $ to do this run, they need the specifics.